app/views/causes/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <h2>Choose a cause you care about</h2>
       <div class="cause-cards">
         <div class="cause-card" id="border">
-          <h3>CAUSE #</h3>
+          <h3><%= @cause1.name %></h3>
```

Comment:
what is the 1 after @cause?

Reply:
Oops. That shouldn't be there. I was testing why something wasn't working as my iteration was on a separate branch. Will push a correction now.